elm/View.elm

```diff
@@ -51,8 +51,13 @@ gameView : GameModel -> Html GameMsg
 gameView model =
     div [ class "game-view" ] <|
         List.concat
-            [ [ topBar model
-              , div [ class "active-state" ]
+            [ [ topBar model ]
+            , -- [note] rounding error?
+              if model.health == 0 then
```

Comment:
So you're showing the death stage when health = 0, but the aliveness is set by the Alive bool below? So what's the authoritative way to tell whether you're dead or not

Reply:
Pls ignore